src/lib/stitching/exchange/v2/stitching.ts

```diff
@@ -349,6 +356,87 @@ export const exchangeStitchingEnvironment = ({
           },
         },
       },
+      Mutation: {
+        createInquiryOffer: {
```

Comment:
Since "offer" can mean 2 different things in Exchange: a "mode" of an order model and a separate model associated with an order. Let's maybe change this to `createInquiryOfferOrder` to be consistent.